### PR TITLE
Removing bashrc env vars and fixing GitHub reference.

### PR DESCRIPTION
**Pull request by t-brown**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: content/03-hpc-aws-parallelcluster-workshop/03-initialize-pc.md [hunk 1]
```diff
@@ -24,17 +24,12 @@ If you have time and are curious, you can examine the different steps of the scr
 curl -O https://raw.githubusercontent.com/aws-samples/awsome-hpc/main/apps/wrf/scripts/setup/SC22_create_parallelcluster_config.sh
 ```
 
-2. Execute the script to retrieve network information.
-```bash
-source ./SC22_create_parallelcluster_config.sh
```
Comment (mhuguesaws) [inline content/03-hpc-aws-parallelcluster-workshop/03-initialize-pc.md:29]: Keep that line

**Review by mhuguesaws — COMMENTED** [review]
Left comments.

File: content/03-hpc-aws-parallelcluster-workshop/03-initialize-pc.md (hunk 1 above)
Comment (t-brown) [reply content/03-hpc-aws-parallelcluster-workshop/03-initialize-pc.md:29]: Doh! Updated.

**Review by mhuguesaws — APPROVED** [review]
LGTM

**Pull request merged**